From: Simon McVittie <smcv@debian.org>
Date: Thu, 31 Mar 2016 09:48:18 +0100
Subject: tests/admin-test.sh: this is a bash script, not a POSIX sh script

The "function foo()" syntax is bash-specific, and Colin indicated in
PR #226 that he prefers to require bash rather than trying to support
every POSIX shell.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/ostreedev/ostree/pull/232
---
 tests/admin-test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/admin-test.sh b/tests/admin-test.sh
index 3a04a69..43dcbb5 100755
--- a/tests/admin-test.sh
+++ b/tests/admin-test.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # Copyright (C) 2011,2014 Colin Walters <walters@verbum.org>
 #
 # This library is free software; you can redistribute it and/or
